frontend/src/components/nodes/ValidatorMetadataRow.tsx

```diff
@@ -0,0 +1,134 @@
+                      target="_blank"
+                    >
+                      {poolWebsite}
+                    </a>
+                  </Col>
+                </Row>
+              </Col>
+            )}
+            {poolEmail && (
+              <Col className="validator-nodes-content-cell" xs="auto">
+                <Row noGutters>
+                  <Col className="validator-nodes-details-title">Email</Col>
+                </Row>
+                <Row noGutters>
+                  <Col className="validator-nodes-text">
+                    <a href={`mailto:${poolEmail}`}>{poolEmail}</a>
+                  </Col>
+                </Row>
+              </Col>
+            )}
+            {poolTwitter && (
+              <Col className="validator-nodes-content-cell" xs="auto">
+                <Row noGutters>
+                  <Col className="validator-nodes-details-title">Twitter</Col>
+                </Row>
+                <Row noGutters>
+                  <Col className="validator-nodes-text">
+                    <a
+                      href={`https://twitter.com/${poolTwitter}`}
+                      rel="noreferrer noopener"
+                      target="_blank"
+                    >
+                      {poolTwitter}
+                    </a>
+                  </Col>
+                </Row>
+              </Col>
+            )}
+            {poolDiscord && (
+              <Col className="validator-nodes-content-cell" xs="auto">
+                <Row noGutters>
+                  <Col className="validator-nodes-details-title">Discord</Col>
+                </Row>
+                <Row noGutters>
+                  <Col className="validator-nodes-text">
+                    <a
+                      href={poolDiscord}
+                      rel="noreferrer noopener"
+                      target="_blank"
+                    >
+                      {poolDiscord}
+                    </a>
+                  </Col>
+                </Row>
+              </Col>
+            )}
+            {poolDescription && (
+              <Col className="validator-nodes-content-cell">
+                <Row noGutters>
+                  <Col className="validator-nodes-details-title">
+                    Description
+                  </Col>
+                </Row>
+                <Row noGutters>
+                  <Col className="validator-nodes-text">
+                    <small>{poolDescription}</small>
+                  </Col>
+                </Row>
+              </Col>
+            )}
+          </>
+        ) : (
+          <Col className="validator-nodes-content-cell">
+            <p className="text-center">
+              If you are node owner feel free to fill all{" "}
+              <a
+                href="https://github.com/zavodil/near-pool-details#description"
+                target="_blank"
+              >
+                data
+              </a>{" "}
+              to promote your own node!
```

Comment:
Same comment about internationalization. From now on, only add text strings to the transactions files.